doc/developers/develop.rst

```diff
@@ -618,16 +618,25 @@ X_types (default=['2darray'])
     ``'categorical'`` data. For now, the test for sparse data do not make use
     of the ``'sparse'`` tag.
 
-
-To override the tags of a child class, one must define the `_more_tags()`
-method and return a dict with the desired tags, e.g::
+It is unlikely that the default values for each tag will suit the needs of your
+specific estimator. Additional tags can be created or default tags can be
```

Comment:
"Additionnal tags can be created"

I thought we agreed not to support that https://github.com/scikit-learn/scikit-learn/pull/18797#discussion_r533403752? (or that's how I interpret @ogrisel's +1)

Reply:
There is a difference between supporting in `_safe_tags` and people creating their owns tags within their library using `_more_tags`. This is a real need here:

https://github.com/rapidsai/cuml/pull/3113/files#diff-e4bd6eee2eca2b0619b03a5f6ba7b471b4ca03080a6619d0079105d5f13c2165R34-R35

We have something similar in imbalanced-learn since the introduction of tags.

Reply:
My +1 was to remove the default param to the `_safe_tags`. I think third-party implementers are free to add other tags in their own estimators if they which. cuML is already doing in in their master branch apparently:

https://github.com/rapidsai/cuml/pull/3113/files